web/satellite/src/components/account/referral/ReferralArea.vue

```diff
@@ -0,0 +1,412 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div class="referral-container">
+		<NoProjectState v-if="isNoProjects"/>
+		<div class="referral-container__wrapper" v-if="!isNoProjects">
+			<p class="referral-container__title">Refer A Friend And You'll Both Get Free Credits</p>
+			<p class="referral-container__subtitle">You Get X GB, They Get X GB</p>
```

Comment:
I think the `X` here should be dynamic and display the value from the active reward

Reply:
Thought its a static value based on organization politics

Reply:
@stoweandi is checking on this one